libraries/BLE/src/BLEDeviceManager.h

```diff
@@ -0,0 +1,421 @@
+/*
+  BLE Device API
+  Copyright (c) 2016 Arduino LLC. All right reserved.
+
```

Comment:
Will move implementation related class definitions/files to the internal subdirectory.